zebra-state/src/error.rs

```diff
@@ -55,6 +55,11 @@ pub enum ValidateContextError {
     #[non_exhaustive]
     NotReadyToBeCommitted,
 
+    #[cfg(feature = "getblocktemplate-rpcs")]
+    #[error("block parent not found in non-finalized best chain")]
```

Comment:
Do we need to mention that it is for a proposal here?

Reply:
Yeah, I think that might be useful context for debugging, added in [`8420b75`](https://github.com/ZcashFoundation/zebra/pull/5870/commits/8420b75013c1a05ea11a9799d1f87bd338c3c27c).